GRIDTHEME.md

```diff
@@ -0,0 +1,102 @@
+GridView Theme
+==============
+
+Currently Gridview is still under extreme development and what is listed in here may
+change or become broken later.  Please be sure to post any problems in this project's
+issues.
+
+
+Quick Overview Guide
+====================
+**Detailed List View MetaData is now available in gridview**
+
+Display current selected game.
+```
+<text name="md_title"> <ALL>
+```
+
+Setup for the grid as a whole.
+```
```

Comment:
You can add the xml modifier to the code-block, if you want to make this fancy: ` ```xml `